pkg/features/system/collector.go

```diff
@@ -369,3 +391,35 @@ func (c *systemCollector) collectSatelites(client *rpc.Client, ch chan<- prometh
 		ch <- prometheus.MustNewConstMetric(hardwareInfoDesc, prometheus.GaugeValue, float64(1), l...)
 	}
 }
+
+func (c *systemCollector) collectLicense(client *rpc.Client, ch chan<- prometheus.Metric, labelValues []string) {
+	r := &licenseInformation{}
+	err := client.RunCommandAndParse("show system license usage", r)
+
+	if err == nil {
+		for i := range r.LicenseInfo.License {
+			licenseLabels := append(labelValues,
+				strings.ToLower(r.LicenseInfo.License[i].Name),
+				strings.ToLower(r.LicenseInfo.License[i].Description))
+
+			ch <- prometheus.MustNewConstMetric(licenseUsedDesc, prometheus.GaugeValue, float64(r.LicenseInfo.License[i].Used), licenseLabels...)
```

Comment:
for better readability: would you mind using a loop variable in the block instead of using the index?